plugins/device/api/req.go

```diff
@@ -10,7 +10,8 @@ const (
 
 // AddDeviceReq structure
 type AddDeviceReq struct {
-	Device string `json:"device"`
+	Device          string `json:"device"`
```

Comment:
can one device be used for both lvm and loop?

Reply:
No. Raw device needs to be registered for LVM.